packages/waku/src/router/client.ts

```diff
@@ -265,8 +272,7 @@ export function Link({
       };
     }
   }, [unstable_prefetchOnView, router, to]);
-  const onClick = (event: MouseEvent<HTMLAnchorElement>) => {
-    event.preventDefault();
```

Comment:
Removing this causes e2e tests to fail, doesn't it?

Reply:
Yes, that was a mistake. I thought I moved it, not deleted.